docs/FunctionsChangelog.md

```diff
@@ -3,14 +3,14 @@
             [def files](/onnx/defs) via [this script](/onnx/defs/gen_doc.py).
             Do not modify directly and instead edit function definitions.*
 # ai.onnx (default)
-## Version 8 of domain ai.onnx (default)
-### <a name="MeanVarianceNormalization-8"></a>**MeanVarianceNormalization-8**</a>
+## Version 9 of domain ai.onnx (default)
```

Comment:
Can we also run the doc comparison in the CI to make sure the doc is up to date?

Reply:
We run the doc_gen and git diff in CI but I think the default domain is not checked (we set to build onnx_ml and thus only -ml files are generated). 
I plan to work on a long-pending PR (which is to merge function and operator.md) files after this is merged. I can enable the script to build default docs no matter onnx_ml is set.

Reply:
@houseroad , good point, however, this doc (Function.md) will be merged into Operator.md, so let's keep it as it is right now. sounds good?

Reply:
@raymondxyang @linkerzhang that sounds good to me, thanks.